engine/lib/entities.php

```diff
@@ -328,7 +329,8 @@ function can_write_to_container($user_guid = 0, $container_guid = 0, $type = 'al
 			array(
 				'container' => $container,
 				'user' => $user,
-				'subtype' => $subtype
+				'subtype' => $subtype,
+				'checking_owner' => ($checking === 'owner'),
```

Comment:
I dunno, I figured boolean would be simpler. Also $checking won't be set anywhere else. I supposed it could default to "container".

`$params['real_container']` is another option, maybe easier to understand.